engine/apps/alerts/signals.py

```diff
@@ -14,6 +14,12 @@
     ]
 )
 
+alert_group_created_signal = django.dispatch.Signal(
```

Comment:
Please, add comment that subscriber to that signal must be super-fast, since it's process of creating alert group